docs/cli/commands/run.md

```diff
@@ -22,5 +22,10 @@ npx cap run [options] <platform>
 - `--list`: Print a list of target devices available to the given platform
 - `--no-sync`: do not run the sync command
 - `--scheme <schemeName>`: set the scheme of the iOS project
+- `--configuration <name>`: Configuration name of the iOS Scheme
 - `--target <id>`: Run on a specific target device
+- `--live-reload`: Enable Live Reload
```

Comment:
`npx cap run --help` shows this as `-l, --live-reload`, if you don't want to show it like that, you can add a new line for `-l` and say "shorthand for --live-reload" or something like that

Reply:
I've added `-l`.